scripts/expected_doxygen_warnings.txt

```diff
@@ -1,40 +1,24 @@
 /cbmc/src/solvers/miniBDD/miniBDD.cpp:402: warning: documented symbol `mini_bddt mini_bddt::operator &' was not declared or defined.
-/cbmc/src/solvers/sat/satcheck_glucose.cpp:228: warning: no matching class member found for 
+/cbmc/src/solvers/sat/satcheck_glucose.cpp:228: warning: no matching class member found for
```

Comment:
Are these whitespaces changes that the second commit reverts?

Reply:
To answer my own questions:
Yes.  Assuming the two commits are squashed, it doesn't look like there will be any remaining unexpected changes.